src/utils/monaco/yql/yql.keywords.ts

```diff
@@ -0,0 +1,13 @@
+export const keywords =
```

Comment:
This data should be moved to separate library. But for now I decided to copy it directly, because it significantly improve view of yql query in editor.